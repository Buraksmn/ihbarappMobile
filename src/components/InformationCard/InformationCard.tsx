import {View, Text, TouchableOpacity} from 'react-native';
import React from 'react';

export default function InformationCard({
  title,
  value,
}: {
  title: string;
  value: string;
}) {
  return (
    <View style={{backgroundColor: '#fad390', padding: 20, marginVertical: 15}}>
      <View
        style={{
          flexDirection: 'row',
          justifyContent: 'space-between',
          marginBottom: 10,
          borderRadius: 10,
        }}>
        <Text style={{fontWeight: '500'}}>{title}</Text>
        <TouchableOpacity>
          <Text style={{fontWeight: '500'}}>X</Text>
        </TouchableOpacity>
      </View>

      <Text style={{fontWeight: '300'}}>{value}</Text>
    </View>
  );
}
