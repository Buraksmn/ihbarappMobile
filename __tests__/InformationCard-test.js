import React from 'react';
import renderer from 'react-test-renderer';
import InformationCard from '../src/components/InformationCard/InformationCard';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <InformationCard title="Test Nedir" value="Test Nedir Açıklaması" />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
