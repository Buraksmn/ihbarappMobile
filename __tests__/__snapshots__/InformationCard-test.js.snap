// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fad390",
      "marginVertical": 15,
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "borderRadius": 10,
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginBottom": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontWeight": "500",
        }
      }
    >
      Test Nedir
    </Text>
    <View
      accessible={true}
      collapsable={false}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "fontWeight": "500",
          }
        }
      >
        X
      </Text>
    </View>
  </View>
  <Text
    style={
      Object {
        "fontWeight": "300",
      }
    }
  >
    Test Nedir Açıklaması
  </Text>
</View>
`;
